frontend/src/stories/components/Rider/RiderTable.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import RiderTable from 'main/components/Rider/RiderTable';
+import { riderFixtures } from 'fixtures/riderFixtures';
+
+export default {
+    title: 'components/Rider/RiderTable',
+    component: RiderTable
+};
+
+const Template = (args) => {
+    return (
+        <RiderTable {...args} />
+    )
+};
+
+export const Empty = Template.bind({});
+
+Empty.args = {
+    ride: []
+};
+
+
+export const ThreeSubjectsNoButtons = Template.bind({});
+
+ThreeSubjectsNoButtons.args = {
+    ride: riderFixtures.threeRidesTable,
+    showButtons: false,
```

Comment:
At least for me, the buttons still show up in this story:

https://ucsb-cs156-s23.github.io/proj-gauchoride-s23-5pm-4/prs/32/storybook/?path=/story/components-rider-ridertable--three-subjects-no-buttons

Reply:
I have changed it so that the buttons showing up is determined based on the role of the user in the latest commit, the show buttons is no longer an attribute.